common/locales/en/defaultTasks.json

```diff
@@ -1,21 +1,19 @@
 {
-  "defaultHabit1Text": "Productive Work (Click the pencil to edit)",
```

Comment:
The other json files are pulled down from Transifex and not edited directly. 

The English ones have 2 spaces because that is our code style. 

They have blank spaces to do some loose organization to make them easier to read. 

The en files that are 4 spaces should be changed to 2, but that's a concern for a separate PR.